cli/src/semgrep/app/scans.py

```diff
@@ -319,6 +321,7 @@ def report_findings(
             "exit_code": 1
             if any(match.is_blocking and not match.is_ignored for match in all_matches)
             else 0,
+            "dependency_parser_errors": [e.to_json() for e in dependency_parser_errors],
```

Comment:
Send as part of `complete` API call

Reply:
This means any parse errors will not be captured if there are Fatal errors running semgrep